example/index.html

```diff
@@ -28,7 +28,6 @@
     }
 
     input {
-      outline: 0;
```

Comment:
It's important to never remove the outline.
https://www.w3.org/TR/WCAG20/#navigation-mechanisms-focus-visible